queries/go.scm

```diff
@@ -222,3 +222,59 @@
     .
   )
 ) @anonymousFunction @namedFunction
+
+;; switch-based branch
+
+(
+  [
+    (default_case)
+    (expression_case)
+    (type_case)
+  ] @branch
+  (#trim-end! @branch)
+  (#insertion-delimiter! @branch "\n")
+)
+
+[
+  (type_switch_statement)
+  (expression_switch_statement)
+] @branch.iteration
+
+;; if-else-based branch
+
+;; first if in an if-else chain
+(
+  (
+    (if_statement
+      consequence: (block) @branch.end.endOf
+    ) @_if
+    (#not-parent-type? @_if if_statement)
+  )
+ @branch.start.startOf
```

Comment:
this is off. I'm not sure if it's an autoformatter bug or not, but I would just change your code this capture right after `@_if` and remove these surrounding parens. not sure what they're accomplishing?

Reply:
done